Sources/FixedParameterFields.swift

```diff
@@ -32,16 +35,18 @@ struct FixedParameterFieldsBuilder: Builder {
     let location: ParameterLocation
     let description: String?
     let required: Bool
+    let example: Any?
 
     init(map: Map) throws {
         name = try map.value("name")
         location = try map.value("in")
         description = try? map.value("description")
         required = (try? map.value("required")) ?? false
+        example = try? map.value("x-example")
```

Comment:
`example` is not a documented key for parameter. "body" parameters can have a `schema` which could have an `example`, but non-"body" parameters do not have an `example` key. [Here's a link to the spec where it discusses parameters](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameter-object).

In order to introduce this feature on all parameter types, I added support for `x-example` to allow specs utilizing the feature to remain valid Swagger OpenAPI 2.0 specs.